rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -126,7 +125,7 @@ def ppo_surrogate_loss(policy, model, dist_class, train_batch):
         train_batch[Postprocessing.ADVANTAGES],
         train_batch[SampleBatch.ACTIONS],
         train_batch[BEHAVIOUR_LOGITS],
```

Comment:
Does it make sense to remove BEHAVIOUR_LOGITS now that we have the action dist out?

Reply:
Absolutely. Will do.

Reply:
Done.